### PR TITLE
Update pkg docs 📝

### PR DESCRIPTION
**Pull request by jonmcalder**
For some reason I happened to spot your recent commits, and given that I'm somehow listed as a pkg co-author for having done little more than contribute some documentation and a hex logo, I figured I better pull my weight. So here are some more documentation updates 🤣 

- Add missing documentation for `fuel_price_usd_exchange_rate` & `fuel_price_brent_crude`
- Add missing LICENSE file
- Add basic pkgdown config (just need to enable GitHub pages and run `usethis::use_github_action("pkgdown")` to setup the relevant GHA)

### CODE REVIEW

File: R/fuel-price-used-exchange-rate.R [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#' Exchange rate history for the South African Rand against the US Dollar.
```
Comment (datawookie) [inline R/fuel-price-used-exchange-rate.R:1]: Please remove period from end of line.

**Review by datawookie — CHANGES_REQUESTED** [review]
Hi John,

Thanks for this. We're actually actively using the data from this package now, hence the renewed enthusiasm.

Thanks very much for your contribution. I've made one minor request (removing a period for consistency).

Also, I'd like to suggest that we set this up to use Travis to build the pkgdown site. I've used this on a few other packages and it's very convenient. If you'd like to put that in place, that'd be cool. Otherwise I'd be happy to hook it up.

Thanks, Andrew.

File: R/fuel-price-used-exchange-rate.R (hunk 1 above)
Comment (jonmcalder) [reply R/fuel-price-used-exchange-rate.R]: Ok sure - some of the existing titles had trailing periods and some were without, so I've removed them across the board for consistency.

**Comment by jonmcalder** [comment]
Nice! I thought you might be making use of it - that's great.

Re setting up the pkgdown site, are you aware that Travis has been acquired by a private company and is discontinuing their free / open source tier? See here for more info: https://ropensci.org/technotes/2020/11/19/moving-away-travis/

Either way, it's your repo and I don't have sufficient permissions so I'll leave it to you to setup the CI.

File: R/fuel-price-used-exchange-rate.R (hunk 1 above)
Comment (datawookie) [reply R/fuel-price-used-exchange-rate.R]: Thanks, @jonmcalder. Yup, this is an artefact of having a few people hacking on this. Thanks for cleaning it all up! :+1:

**Comment by datawookie** [comment]
Yup, I'm aware of the issues with Travis. Going to take the route of least resistance for the moment but certainly will look at a more durable approach later on.

**Pull request merged**